api/mock/facade.go

```diff
@@ -83,7 +83,7 @@ func (f *Facade) GetProof(rootHash string, address string) (*shared.GetProofResp
 }
 
 // GetProofCurrentRootHash -
-func (f *Facade) GetProofCurrentRootHash(address string) (*shared.GetProofResponse, error) {
+func (f *Facade) GetProofCurrentRootHash(address string) (*common.GetProofResponse, error) {
```

Comment:
I think this is a good time to rename this struct and file to `FacadeStub`

Reply:
Done.